applications/nrf_desktop/doc/settings_loader.rst

```diff
@@ -33,7 +33,11 @@ For details on the default configuration alignment, see the following sections.
 Settings backend
 ================
 
-By default, nRF Desktop devices use non-volatile storage settings backend (:kconfig:option:`CONFIG_SETTINGS_NVS`).
+By default, the nRF Desktop application, depending on the non-volatile memory technology used by the device, uses one of the following settings backends:
+
+* :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` - Used for the devices with non-volatile memory that does not require explicit erase (MRAM, RRAM).
```

Comment:
*that do not

Reply:
"devices that do not" or "memory that does not"?
Isn't it the memory that does not require explicit erase?

Reply:
it's devices really, the memory doesn't need erase but that's because of the device e.g. external QSPI flash or internal RRAM